lib/messagebird/conversation.rb

```diff
@@ -0,0 +1,34 @@
+require 'messagebird/base'
+require 'messagebird/contact'
+require 'messagebird/conversation_channel'
+
+module MessageBird 
+  class Conversation < MessageBird::Base
+    attr_accessor :id, :contact, :channels, :status, :messages,
```

Comment:
I'm not really into Ruby anymore, but does it make sense to define `contact`, `channels` and others to which you've added setters as `attr_reader`? Maybe @marcelcorso or @epels  have an opinion on that?